client/flutter/lib/pages/protect_yourself.dart

```diff
@@ -40,18 +40,28 @@ Text _message(String input) {
     }
 
     spans.add(
-      TextSpan(text: value, style: bold),
+      TextSpan(
+          text: value,
+          style: TextStyle(
+            fontWeight: FontWeight.bold,
+            color: Color(0xff3C4245),
+            fontSize: 16,
+          )),
     );
     before = match.end;
   }
 
   spans.add(
     TextSpan(
-      text: input.substring(before),
-    ),
+        text: input.substring(before),
+        style: TextStyle(
+          color: Color(0xff3C4245),
+          fontSize: 16,
+        )),
   );
   return Text.rich(
-    TextSpan(style: normal, children: spans),
+    TextSpan(style: TextStyle(
+    ), children: spans),
```

Comment:
This is an empty `TextStyle`? Should it not also be like the others?

Reply:
@PieterAelse sorry, I had a mistake, I fix it and add it. No, it would not be necessary considering the objectives of the issue

Reply:
@PieterAelse would you like me  to do any other changes in the code or fixes?